public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -166,6 +176,13 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
     }
   }
 
+  getInfluxTargetSignature(request: DataQueryRequest<InfluxQuery>, targ: InfluxQuery) {
```

Comment:
@itsmylife what uniquely defines a query for influx?

Reply:
I think we do not have a way to do this for influx.

Reply:
In prometheus we use the query interval, the interpolated expression, exemplar status etc, I think with influx we'll just need to either echo out the interpolated query string (if raw query mode), or the JSON encoding of the select `InfluxQueryPart[][]` concatenated with the interval. Basically we're trying to figure out which properties of an influxQuery change when the query changes

Reply:
@itsmylife I noticed that basically all of the props change the query, so I went ahead and added basically all of these props. I wasn't sure about the `adhocFilters`or `resultFormat`, do these change the backend response?